src/commands/inlineVerify.js

```diff
@@ -43,17 +43,25 @@ const inlineVerify = async (certfiles, verificationsOut) => {
         count++;
         const signature = await s.signature;
         const timestamp = utils.format_date(signature.packets[0].created);
+        let found = false;
         for (const cert of verificationKeys) {
-          const signingKey = await cert.getSigningKey(s.keyID, null).catch(() => null);
+          const [signingKey] = await cert.getKeys(s.keyID);
           if (signingKey) {
             verifications +=
               timestamp
                 + ' ' + signingKey.getFingerprint().toUpperCase()
                 + ' ' + cert.getFingerprint().toUpperCase()
                 + '\n';
+            found = true;
             break;
           }
         }
+        if (!found) {
```

Comment:
How can this case happen (bugs aside)?

Reply:
It can't, really, I added it just in case. I can also remove it again.